### PR TITLE
AB#3385  configuration for backend project with teamcity

### PR DESCRIPTION
**Pull request by ESDC-Olivier-Roy**
### Description
added the configuration to have the backend project deployed in the cluster dev environment. There are changes in the files of this PR and on teamcity

### Related Azure Boards Work Items
[AB#3385](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3385) 
-->

### Checklist

- [x] I have tested the changes locally
- [x] I have updated the documentation if necessary
- [ ] I have added/updated tests that prove my fix is effective or that my feature works
- [ ] I have checked that my code follows the project's coding style by running `npm run format:check`
- [ ] I have checked that my code contains no linting errors by running `npm run lint`
- [ ] I have checked that my code contains no type errors by running `npm run typecheck`
- [ ] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [ ] I have checked that all e2e tests pass by running `npm run test:e2e`

### CODE REVIEW

File: gitops/base/backend/deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          ports:
+            - name: http
+              containerPort: 8080
+          #livenessProbe:
+          #  tcpSocket:
+          #    port: http
+          #readinessProbe:
+          #  httpGet:
+          #    path: /api/readyz
+          #    port: http
```
Comment (gregory-j-baker) [inline gitops/base/backend/deployments.yaml:29]:
```suggestion
          livenessProbe:
            httpGet:
              path: /actuator/health/liveness
              port: http
            initialDelaySeconds: 30
          readinessProbe:
            httpGet:
              path: /actuator/health/readiness
              port: http
            initialDelaySeconds: 30
```

The API is configured with kubernetes liveness and readiness probes, so you should use them here.

File: gitops/base/backend/deployments.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+              cpu: 500m
+              memory: 1024Mi
+          securityContext:
+            allowPrivilegeEscalation: false
+          volumeMounts:
+            - name: audit-logs
+              mountPath: /home/node/logs
+      volumes:
+        - name: audit-logs
+          emptyDir: {}
```
Comment (gregory-j-baker) [inline gitops/base/backend/deployments.yaml:44]: These lines should be removed.

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]

File: gitops/base/backend/deployments.yaml (hunk 1 above)
Comment (ESDC-Olivier-Roy) [reply gitops/base/backend/deployments.yaml]: done

File: gitops/base/backend/deployments.yaml (hunk 2 above)
Comment (ESDC-Olivier-Roy) [reply gitops/base/backend/deployments.yaml]: removed

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**